core/embed/upymod/modtrezorio/modtrezorio-webusb.h

```diff
@@ -127,10 +127,25 @@ STATIC mp_obj_t mod_trezorio_WebUSB_write(mp_obj_t self, mp_obj_t msg) {
 STATIC MP_DEFINE_CONST_FUN_OBJ_2(mod_trezorio_WebUSB_write_obj,
                                  mod_trezorio_WebUSB_write);
 
+/// def read(self, buf: bytes) -> int:
```

Comment:
we want the following signature: 
```suggestion
/// def read(self, buf: bytes, offset: int = 0, limit: int | None = None) -> int:
```

Reply:
fixed in https://github.com/trezor/trezor-firmware/pull/4419/commits/34559a47716ad0eb41603d66ac4ebe9ed1e6841c and https://github.com/trezor/trezor-firmware/pull/4419/commits/51556ef6ba7632ecda7feb7e9e816dea1ce02d09